packages/compass-settings/src/components/settings/theme.tsx

```diff
@@ -120,10 +135,12 @@ export const ThemeSettings: React.FunctionComponent<ThemeSettingsProps> = ({
           id="theme-selector"
           onChange={handleSelectorChange}
           value={themeValue}
+          size="full"
```

Comment:
There's compact, default and full. Full leaves enough space for the image and text. I think we'll have the best control if we just don't have separate text or integrate it into the image.